.github/ref/OM4.single_column.COBALT.p4/docker-linux-gnu/ocean.stats

```diff
@@ -1,11 +1,11 @@
-  Step,       Day,  Truncs,      Energy/Mass,      Maximum CFL,  Mean Sea Level,  Total Mass,  Mean Salin, Mean Temp, Frac Mass Err,   Salin Err,    Temp Err
+   Step,       Day,  Truncs,      Energy/Mass,      Maximum CFL,  Mean Sea Level,  Total Mass,  Mean Salin, Mean Temp, Frac Mass Err,   Salin Err,    Temp Err
```

Comment:
@feida6996, I don't want to be so picky, but the RT test crashed and complained about the additional white space in the first line. I believe it will pass if you remove it